src/App.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { Repos } from "./components";
+import { RepoContextProvider } from "./context/RepoContext";
+import { Header } from './layout/Header'; 
+import { Footer } from './layout/Footer';
+
+function App() {
+  return (
+    <RepoContextProvider>
```

Comment:
What is RepoContextProvider and how does it work? Just curious because we've not seen it before!